torch_frame/data/mapper.py

```diff
@@ -259,7 +259,8 @@ def forward(
             torch.from_numpy(
                 ser.dt.second.values).to(device=device).unsqueeze(1)
         ]
-        return torch.stack(tensors).permute(1, 2, 0).to(torch.float32)
+        return torch.stack(tensors).permute(1, 2,
+                                            0).squeeze(1).to(torch.float32)
```

Comment:
Why convert to float?

Reply:
I think the values by default is `float64`

Reply:
I would prefer to keep the materialized data as integer, as they are int based, and we don't have to deal with floating point precision issues.